### PR TITLE
Backport-2.4-980 New content for v. 2.4 on LDAP configuration for private automation hub…

### PR DESCRIPTION
**Pull request by jself-sudoku**
Added new reference module w/ two sets of instructions (discrete headings):

- LDAP configuration on private automation hub (ref-ldap-config-on-pah.adoc)

     -   Setting up your inventory file variables
     - Configuring extra LDAP parameters 

- Updated assembly-platform-install-scenario.adoc to include xref and the include:: for ref-ldap-config-on-pah.adoc.

### CODE REVIEW

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 1]
```diff
@@ -0,0 +1,177 @@
+:_content-type: REFERENCE
+
+[id="ref-ldap-config-on-pah_{context}"]
+= LDAP configuration on {PrivateHubName}
+
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
```
Comment (hherbly) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:9]: Is "uri" right here, or should it be "url"? Ignore if correct.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+When you configure your {PrivateHubName} with LDAP authentication, you must set the proper variables in your inventory files during the installation process.
+
+//.Prerequisites (Add Prereqs to 2.2 only)
+
+//* Ensure that your system is running {PlatformName} 2.2.1 or later.
+//* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
```
Comment (hherbly) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: hhttps in the URL--extra "h"

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:9]: Yes, that's how it should be.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
+* `automationhub_ldap_bind_dn`
+* `automationhub_ldap_bind_password`
+* `automationhub_ldap_user_search_base_dn`
+* `automationhub_ldap_group_search_base_dn`
+
+If any of these variables are missing, the Ansible Automation installer will  not complete the installation.
```
[the remark this answers is not in the document]
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:15]: There's actually just one space. The font makes it look like two spaces. This was fixed in the backport, but when  you have to backport to another version repo (e.g. 2.4), as far as I know, you have to create the backport using the commit # from the original PR. Your note reminds me to check the first backport to 2.3 to make sure we have covered typos like this one.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 2 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: Oops! That was corrected in the first backport (to 2.3). Will correct here as well.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
+
+. Use the following example as a guide to set up your {PlatformNameShort} inventory file:
++
+-----
+automationhub_authentication_backend = "ldap"
+
+automationhub_ldap_server_uri = "ldap://ldap:389"   (for LDAPs use  automationhub_ldap_server_uri = "ldaps://ldap-server-fqdn")
```
[the remark this answers is not in the document]
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:36]: STET (same as above)

**Pull request merged**